front/poke/temporal/activities.ts

```diff
@@ -297,7 +302,35 @@ export async function deleteAgentsActivity({
         },
         transaction: t,
       });
-      // TODO(@fontanierh): missing tables query here.
+      const tablesQueryConfigurations =
+        await AgentTablesQueryConfiguration.findAll({
+          where: {
+            agentConfigurationId: agent.id,
+          },
+          transaction: t,
+        });
+      await AgentTablesQueryAction.destroy({
+        where: {
+          tablesQueryConfigurationId: {
```

Comment:
here you are using ids and on the next destroy you are using model ids, is that expected?

Reply:
This one is just linked by sId (no foreign key)

Reply:
ok ok, just wanted to make sure 👍